core/deployment/src/main/java/io/quarkus/deployment/dev/RuntimeUpdatesProcessor.java

```diff
@@ -848,10 +912,10 @@ public static void setLastStartIndex(IndexView lastStartIndex) {
 
     @Override
     public void close() throws IOException {
-        if (timer != null) {
```

Comment:
Shouldn't close() also cancel the Timer if used?
```java
if (testClassChangeTimer != null) {
    testClassChangeTimer.cancel();
}
```

Reply:
oops, I will fix

Reply:
fixed